README.md

```diff
@@ -0,0 +1,72 @@
+# SteeltoeOSS Docker Images
+
+GitHub repo for server images to use for local development with SteeltoeOSS.
+
+## Building
+
+### Pre-Requisites
+
+The following tools are required to build any image in this repository:
+
+1. PowerShell or pwsh
+1. Docker or Podman
```

Comment:
The script fails when Docker is unavailable. Was it tested with Podman, or should it just not be mentioned here?